src/index.html

```diff
@@ -13,5 +13,154 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="http://localhost:8080/" method="post">
```

Comment:
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
    
```
Please, read the task
On form submit send form data to https://mate-academy-form-lesson.herokuapp.com/create-application